components/loginscreen/Loginscreen.module.css

```diff
@@ -0,0 +1,13 @@
+.background {
+  background: var(--main-background-color);
+  display: grid;
+  justify-items: center;
+  height: 100vh;
+  grid-template-rows: 20% 20%;
+}
+.headline {
+  font-family: "AlmondCream", sans-serif;
+  color: #707070;
```

Comment:
Are you going to use this color regularly?
If (true) {
make it a variable in your global css
} else {
code is fine
}

Reply:
> Are you going to use this color regularly?
> If (true) {
> make it a variable in your global css
> } else {
> code is fine
> }

oh maybe you are right, seems it may my h1 font color, thanks :-)